namespaces/blueprint-parse-result.md

```diff
@@ -0,0 +1,119 @@
+# Blueprint Parser Result Namespace
+
+This document defines elements needed for holding the result of parsing
+API Blueprint. This document uses [API Namespace][].
+
+## Content
+
+<!-- TOC depth:2 withLinks:1 updateOnSave:1 -->
+<!-- /TOC -->
+
+## About this Document
+
+This document conforms to [RFC 2119][], which says:
+
+> The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+[MSON][] is used throughout this document.
+
+## Parse Result (Element)
+
+A result of parsing an API Blueprint document.
+
+### Properties
+
+- `element`: parseResult (string, fixed)
+- `content` (array)
+    - (Category)
+    - (Annotation)
+
+### Example
+
+Given following blueprint:
+
+```apib
+# GET /1
+```
+
+The parse result is (using null in `category` content for simplicity):
+
+```json
+["parseResult", {}, {},
+  ["category", {"class": "api"}, {"sourceMap": [[0,9]]}, null],
+  ["annotation", {"class": "warning"}, { "code": 6, "sourceMap": [[0,9]] }, "action is missing a response"]
```

Comment:
Shouldn't the source maps be elements? Here they are plain arrays while below a new element type is defined. Am I missing something?